README.md

```diff
@@ -23,29 +23,32 @@ Parsers are ordered and takes specific input to generate specific output. This w
 
 By using parsers, you dictate the way you receive the data from Specify to fit your own needs.
 
-## Creating your own parser
+## How to create your own parser?
+
+Let's say you want to create a parser named `my-parser`.
 
 1. Git clone this repo
```

Comment:
Add the fork step at first.
1. Fork the current repository
....

Reply:
Done